src/EditorFeatures/CSharpTest/Diagnostics/Iterator/AddYieldTests.cs

```diff
@@ -333,7 +333,9 @@ static IEnumerator<IList<DateTime>> M()
             await TestInRegularAndScriptAsync(initial, expected);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsChangeToYield)]
+        // PROTOTYPE(NullableReferenceTypes): Assert in CustomModifierUtils.CopyMethodCustomModifiers
```

Comment:
>// PROTOTYPE(NullableReferenceTypes): Assert in CustomModifierUtils.CopyMethodCustomModifiers [](start = 8, length = 93)

We should adjust the assert accordingly instead, or make other appropriate adjustments.

Reply:
Will do so when addressing the `// PROTOTYPE` comment.

---
In reply to: [171141725](https://github.com/dotnet/roslyn/pull/25087#discussion_r171141725) [](ancestors = 171141725)